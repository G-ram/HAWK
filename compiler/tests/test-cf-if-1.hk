BEGIN{
    i = 0;
    if(i < 0){
        baz = 1 + 1;
        print(baz);
    }
    print("I should be here");
}
END{}
