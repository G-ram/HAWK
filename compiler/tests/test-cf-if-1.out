BEGIN {
i = 0;
if(i < 0){
baz = 1 + 1;

}else{

}

}
END{

}
